regulations/static/regulations/css/less/module/drawer-content.less

```diff
@@ -34,20 +34,31 @@ This contains all of the styles specific to the TOC
 
       // to account for the last child padding
       // when there's a nested list
-      li ol li:first-child {
-        a {
-          padding-top: 0;
+      &>li > ol > li {
```

Comment:
This looks a bit difficult to figure out what it's targeting... what is it trying to target?